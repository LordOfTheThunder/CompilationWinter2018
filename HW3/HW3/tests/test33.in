struct potato {
	int x;
};
void TestCreatePotato() {
	struct potato t;
	int rr;
	rr = t.x;
}

void main() { print("Max is cool"); }
