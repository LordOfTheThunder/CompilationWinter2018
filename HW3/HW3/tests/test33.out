---end scope---
t struct potato 0
rr INT 1
---end scope---
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
TestCreatePotato ()->VOID 0
main ()->VOID 0
struct potato {INT x}
